src/discordEvents/messageCreate.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    name: 'messageCreate',
+    async execute(client, message) {
+        let instance = client.readInstanceFile(message.guild.id);
+        let channel = instance.channelId.ingamechat
+        if (message.channelId != channel) return;
+        if (message.author.bot) return;
+        let rustplus = client.rustplusInstances[message.guild.id];
```

Comment:
Add a check if rustplus instance even exist? Because if it does not and someone writes in that channel, the bot will crash